src/main/java/com/checkmarx/flow/utils/MarkDownHelper.java

```diff
@@ -0,0 +1,133 @@
+package com.checkmarx.flow.utils;
+
+import com.checkmarx.flow.exception.MachinaRuntimeException;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public class MarkDownHelper {
+
+    private static final String SAST_SCANNER = "SAST";
+    private static final String AST_SAST_SCANNER = "AST-" + SAST_SCANNER;
+    private static final String SCA_SCANNER = "SCA";
+
+    public static final String SCAN_SUMMARY_DETAILS = "Scan Summary & Details";
+    private static final String CX_PREFIX = "Cx-";
+    public static final String AST_DETAILS_HEADER = CX_PREFIX + AST_SAST_SCANNER + " Details";
+
+    static final String MORE_DETAILS_LINK_HEADER = "View more details on Checkmarx UI";
+    static final String SAST_DETAILS_HEADER = CX_PREFIX + SAST_SCANNER + " Details";
+
+    private static final String SUMMARY_SUFFIX = " Summary";
+    public static final String SAST_SUMMARY_HEADER = CX_PREFIX + SAST_SCANNER + SUMMARY_SUFFIX;
+    static final String AST_SUMMARY_HEADER = CX_PREFIX + AST_SAST_SCANNER + SUMMARY_SUFFIX;
+    static final String SCA_SUMMARY_HEADER = CX_PREFIX + SCA_SCANNER + SUMMARY_SUFFIX;
+
+    static final String LINE_BREAK = "<br />";
+    static final String NBSP = "&nbsp;";
+
+    private static final String IMAGE_TEMPLATE = "![%s](%s)";
```

Comment:
@NimrodGolan , @ofersk
Why does the image template need to have the url suffix ( '(%s)' )
it makes the icon be a link - and it doesn't seem to be right

Reply:
in MD you add an image using this template:
`![alt text](http://url/to/img.png)`

in GitHub there is a non standard way, to add a local image using:
`[[url/to/img.png]]`

Reply:
@AvivCX - this is a basic MD for loading an image. where do you see this bug?

Reply:
in the current shape you can press it like a button, and it opens a separate page with the icon.
I dont think it is purposed

Reply:
@NimrodGolan  in GitHub append ?raw=true to the image url will make the trick.
ex: `![](images/logo.png?raw=true)`

this is a GitHub "feature"